src/main/java/io/snyk/jenkins/SnykStepBuilder.java

```diff
@@ -236,7 +235,7 @@ public ListBoxModel doFillSnykTokenIdItems(@AncestorInPath Item item, @QueryPara
         }
       }
       return model.includeEmptyValue()
-                  .includeAs(ACL.SYSTEM, item, SnykApiToken.class)
+                  .includeAs(ACL.SYSTEM2, item, SnykApiToken.class)
```

Comment:
Jenkins 2.479.x introduces Spring Security 6. The former field was deprecated and replaced by `SYSTEM2`.